### PR TITLE
Update Lookup Table schema with a new field

### PR DESCRIPTION
**Pull request by dashaaa**
### Background

Now users can upload lookup table content by specifying it in the FileName field. The DataFile field were a breadcrumbs from previous implementations which is not included here. Supporting of new file extensions is added as well (WIP)

### Changes

* Add a new field to the schema (instead of an obsolete one)

### Testing

* Unit test is edited to match code changes 
* I checked manually that uploading works

### CODE REVIEW

File: panther_analysis_tool/schemas.py [hunk 1]
```diff
@@ -204,7 +204,7 @@
         "AnalysisType": Or("lookup_table"),
         "LookupName": str,
         "Enabled": bool,
-        Or("DataFile", "Refresh"): Or(
+        Or("FileName", "Refresh"): Or(
```
Comment (darwayne) [inline panther_analysis_tool/schemas.py:207]: not clear on what the implication are around this. When you're free can we have a quick chat in slack/zoom?

**Comment by darwayne** [comment]
image

**Comment by darwayne** [comment]
as discussed in our offline chat, looks like one more piece that's needed is to make sure PAT zips the file you expect it to

**Comment by dashaaa** [comment]
> 

Changing FileName -> Filename makes PAT to zip file that is mentioned in this field (similarly to .py files), so the last comment is taken care of

**Comment by dashaaa** [comment]
@darwayne do you mind to take another look?

**Comment by darwayne** [comment]
@dashaaa will do, code looks good, have you done any manual testing to verify this works as expected?

**Comment by dashaaa** [comment]
@darwayne Yes! I was able to upload lookup table yml + data. It works when I do it from the directory itself with `panther_analysis_tool --debug upload --path ./`. I was not able to do it from another directory though, but I am not clear on how it is supposed to work. Please, advise if you know.

**Comment by darwayne** [comment]
that sounds like a [known issue](https://app.asana.com/0/1200908948600035/1202486395033678/f)

**Review by darwayne — APPROVED** [review]

**Pull request merged**